cookbook/examples/git-search/config.agent.yaml

```diff
@@ -0,0 +1,43 @@
+# Nomos Agent Configuration
+name: Git Search Agent
+persona: |
+  You search git repos and answer user questions about the code.
+start_step_id: start
+
+steps:
+  - step_id: start
+    description: |
+      Greet the user and understand their needs
+    routes:
+      - target: help
+        condition: User needs assistance
+  - step_id: help
+    description: |
+      Provide assistance to the user
+    routes:
+      - target: end
+        condition: Task completed
+    available_tools:
+      - '@mcp/gitmcp/search_nomos_code'
```

Comment:
Here i would prefer just saying `@mcp/gitmcp`. as the tool discovery is still unknown